doc/classes/Array.xml

```diff
@@ -441,12 +441,23 @@
 			<return type="Variant" />
 			<description>
 				Returns the maximum value contained in the array if all elements are of comparable types. If the elements can't be compared, [code]null[/code] is returned.
+				To find the maximum value using a custom comparator, use [method reduce] like this (this returns the first maximum value if multiple values would be considered the maximum, as in the example):
+				[codeblock]
+				func _ready():
+				    var arr = [Vector2(0, 1), Vector2(2, 0), Vector2(1, 1), Vector2(1, 0), Vector2(0, 2)]
+				    # In this example we compare the lengths.
+				    print(arr.reduce(func(max, val): return val if cmp_len(val, max) else max))
+
+				func cmp_len(a, b):
```

Comment:
```suggestion
				    print(arr.reduce(func(max, val): return val if is_length_greater(val, max) else max))

				func is_length_greater(a, b):
```

Prefer full names for methods. We can also make it more explicit here, by naming it after what it does. (It also helps with reading the ternary expression).